.golangci.yml

```diff
@@ -0,0 +1,17 @@
+# test-golangci-lint configuration file for golangci-lint
+
+run:
+  skip-files:
+    - lang/lexer.nn.go
+    - lang/y.go
+    - bindata/bindata.go
```

Comment:
This is/will be removed in `https://github.com/purpleidea/mgmt/pull/671` so if that's merged first, this will need to be rebased and updated